src/Command/PermissionsMakeAdminCommand.php

```diff
@@ -51,7 +53,26 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+
         $discordUserId = $input->getArgument('discord_user_id');
+        if (null === $discordUserId) {
+            /** @var UserEntity[] $allUsers */
+            $allUsers = $this->userEntityRepository->findAll();
+            $allUsersNames = array_map(function (UserEntity $x) {
+                return sprintf('%d (%s)', $x->getExternalId(), $x->getUsername());
+            }, $allUsers);
+
+            $helper = $this->getHelper('question');
+            $question = new ChoiceQuestion('Please select user from the list', $allUsersNames);
+
+            $response = $helper->ask($input, $output, $question);
+            if (!$response) {
+                return 1;
+            }
+
+            $discordUserId = substr($response, 0, 18);
```

Comment:
It would be great if we could just reuse response (index) entered by the user to access user entity from already fetched list and stop command execution here by calling some method like `setAdminPermissions(UserEntity $userEntity): void` which sets permissions and persists changes.

We can do that by using normalizer which takes raw user input (index in this case):
https://symfony.com/doc/current/components/console/helpers/questionhelper.html#normalizing-the-answer
(Maybe there is also some other way to do it)

This would also allow us to split execution paths of this command to two:
* when user passed optional parameters and we need to validate input and query database
* when user skipped the parameter to select from the list of users and we can skip validations because we are already working with entities

both resulting in calling mentioned `setAdminPermissions` method at the end when we have user entity.